components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -293,3 +294,47 @@ export function refreshSitePermissions () {
     dispatch(WalletActions.setSitePermissions({ accounts: accountsWithPermission }))
   }
 }
+
```

Comment:
nit: both these functions don't need to be added to an ever-expanding generic `async/lib.ts`, they could be added to a new e.g. `async/hardware.ts` and corresponding `hardware.test.ts`

Reply:
done